convert2rhel/repo.py

```diff
@@ -20,15 +20,21 @@
 import shutil
 
 from convert2rhel.systeminfo import system_info
-from convert2rhel.utils import BACKUP_DIR
+from convert2rhel.utils import BACKUP_DIR, DATA_DIR
 
 
 loggerinst = logging.getLogger(__name__)
 
 
 def get_rhel_repoids():
     """Get IDs of the Red Hat CDN repositories that correspond to the current system."""
-    repos_needed = system_info.default_rhsm_repoids
+
+    reposdir = get_eus_repos_available()
+
+    if not reposdir:
```

Comment:
```suggestion
    EUS_MINOR_VERSIONS = [8.2, 8.4 ,8.6, 8.8, 8.10]  # this can be moved to systeminfo.py

    os_version = ".".join(map(str, system_info.version))
    if os_version not in EUS_MINOR_VERSIONS:
```